src/main/java/uk/gov/hmcts/reform/professionalapi/service/impl/PaymentAccountServiceImpl.java

```diff
@@ -84,7 +84,12 @@ public void deleteUserAccountMaps(Organisation organisation) {
         List<UserAccountMapId> accountsToDelete = generateListOfAccountsToDelete(user, paymentAccount);
 
         userAccountMapService.deleteByUserAccountMapIdIn(accountsToDelete);
+    }
 
+    @Transactional
+    public void editPaymentAccountsForAnOrganisation(PbaEditRequest pbaEditRequest, Organisation organisation) {

```

Comment:
you should remove @Trabsactional from deletePaymentAccountsFromOrganisation and addPaymentAccountsToOrganisation methods

Reply:
Okay I will make this change